up-l1/mqtt_5.adoc

```diff
@@ -235,5 +235,10 @@ When MQTT5 (broker) is used for D2D communication, the topic shall consist of on
 ===== Sending a Message
 `UTransport::send(UMessage)` translates to the MQTT publish to topic `{UMessage.source.authority_name}/{UMessage.sink.authority_name}`
 
+== MQTT5 Connection handling
 
+=== Establishing a connection
+Establishing a connection to an MQTT broker *SHOULD* be done with `cleanSession = true` and `sessionExpiry = 0`. With those settings the broker will not buffer any messages while a the client is offline and will loose any subscriptions on a client disconnect. 
```

Comment:
Please add the OFT tags like above for your requirements.

Reply:
Done via: 

[](https://github.com/jjj-vtm/up-spec/blob/d984eb005d26b15437f95337ada3c953c7687c09/up-l1/mqtt_5.adoc?plain=1#L240C1-L244C1)